### PR TITLE
Fix hypersurface plotting and several other things

### PR DESCRIPTION
**Pull request by shreyasbapat**
I don't know which issue it solves.

### CODE REVIEW

File: src/einsteinpy/hypersurface/schwarzschildembedding.py [hunk 1]
```diff
@@ -174,34 +172,3 @@ def get_values_surface(self, alpha):
 
         Z.reshape((X.shape[0], X.shape[1]))
         return X, Y, Z
-
-    def plot_hypersurface(self, plot_type="wireframe", alpha=100):
```
Comment (ritzvik) [inline src/einsteinpy/hypersurface/schwarzschildembedding.py]: Preserve the function and change the code internally to get results from the plotting module.
We don't want to break APIs. We can put a `DeprecationWarning`

File: src/einsteinpy/hypersurface/schwarzschildembedding.py [hunk 2]
```diff
@@ -174,34 +172,3 @@ def get_values_surface(self, alpha):
...
-        Z = Z.reshape((shape_tuple[0], shape_tuple[1]))
-        if plot_type == "wireframe":
-            ax.plot_wireframe(X, Y, Z, color="black")
-        elif plot_type == "surface":
-            ax.plot_surface(
-                X, Y, Z, rstride=1, cstride=1, cmap="cubehelix", edgecolor="none"
-            )
-
-    def show(self):
-        """
```
Comment (ritzvik) [inline src/einsteinpy/hypersurface/schwarzschildembedding.py]: Same for this.

**Comment by ritzvik** [comment]
And one more thing. Set Matplotlib environment variable as "Agg" in github actions `yml` file. I forgot to do that.

File: src/einsteinpy/hypersurface/schwarzschildembedding.py (hunk 1 above)
Comment (shreyasbapat) [reply src/einsteinpy/hypersurface/schwarzschildembedding.py]: This new 0.3 is all about API breaks!

**Comment by shreyasbapat** [comment]
> And one more thing. Set Matplotlib environment variable as "Agg" in github actions `yml` file. I forgot to do that.

Sure, will do that

File: src/einsteinpy/hypersurface/schwarzschildembedding.py (hunk 1 above)
Comment (ritzvik) [reply src/einsteinpy/hypersurface/schwarzschildembedding.py]: Now that we have decided to release `0.2.2`, please take care of this. 🙏

Comment (shreyasbapat) [reply src/einsteinpy/hypersurface/schwarzschildembedding.py]: Alright. Without my heart to it, but I will take care of that!

**Comment by shreyasbapat** [comment]
Woah, why is symbolic showing up here? ;P

**Comment by shreyasbapat** [comment]
> Woah, why is symbolic showing up here? ;P

Nevermind. I had a bad rebase.

**Comment by shreyasbapat** [comment]
Alright. I will merge this with promise to add tests for Bhavya's code in next Pull request :p (I guess we shouldn't have merged Bhavya's code without tests in the first place)

**Pull request merged**